frontend/src/App.css

```diff
@@ -34,4 +34,244 @@ body {
     margin: 0 auto;
     background: white;
     padding: 10px;
-  }
\ No newline at end of file
+} 
+
+*/
+
+/**
+ * Reset button styles.
+ * It takes a bit of work to achieve a neutral look.
+ */
+ button {
+  padding: 0;
+  border: none;
+  font: inherit;
+  color: inherit;
+  background-color: transparent;
+  /* show a hand cursor on hover; some argue that we
+  should keep the default arrow cursor for buttons */
+  cursor: pointer;
+}
+
+.App {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  flex-basis: 100%;
+}
+
+.course-view {
+  display: flex;
+  overflow: hidden; 
+}
+
+.curri-button {
```

Comment:
We need to change the style names to fit the application better. Also clear out unnecessary stylings.